app/scripts/controllers/transactions.js

```diff
@@ -201,6 +210,7 @@ module.exports = class TransactionController extends EventEmitter {
       nonceLock.releaseLock()
     } catch (err) {
       this.setTxStatusFailed(txId, {
```

Comment:
this is kind of silly, we should have just been able to pass in the err, but that bad idea predates this PR